crates/rpc/src/method/get_class_at.rs

```diff
@@ -76,7 +76,10 @@ pub async fn get_class_at(
             None
         };
 
-        let block_id = input.block_id.to_finalized_coerced();
+        let block_id = input
+            .block_id
+            .to_common_or_panic(&tx)
```

Comment:
panic is wrong here, no? `input.block_id` could be `Pending`, as checked above when initializing `pending_class_hash`, and that check does not evade the initialization of `block_id`...

Reply:
https://github.com/eqlabs/pathfinder/pull/2856/commits/082b4f6cc0800fdc58b29ea49d8f9d415b0940e5